examples/templates/aws-linux/main.tf

```diff
@@ -101,7 +101,7 @@ Content-Transfer-Encoding: 7bit
 Content-Disposition: attachment; filename="userdata.txt"
 
 #!/bin/bash
-sudo -E -u ubuntu sh -c '${coder_agent.dev.init_script}'
+sudo -u ubuntu sh -c '${coder_agent.dev.init_script}'
```

Comment:
We could consider using `-H` to ensure HOME is set.

Reply:
man page says that -H might be a system default, and in ubuntu is seems to be the case. Feels like overkill